<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>293</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Credential Saver</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>D:/Downloads/login.ico</normaloff>D:/Downloads/login.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #191919;
font-family: Montserrat;</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="le_service">
       <property name="styleSheet">
        <string notr="true">color: #F5E8C7;
font-size: 12pt;
margin: 5%;
padding: 10%;
border: 1px solid #404258;
border-radius: 10px;

</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Service name...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_login">
       <property name="styleSheet">
        <string notr="true">color: #F5E8C7;
font-size: 12pt;
margin: 5%;
padding: 10%;
border: 1px solid #404258;
border-radius: 10px;
</string>
       </property>
       <property name="placeholderText">
        <string>Login...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_password">
       <property name="styleSheet">
        <string notr="true">color: #F5E8C7;
font-size: 12pt;
margin: 5%;
padding: 10%;
border: 1px solid #404258;
border-radius: 10px;
</string>
       </property>
       <property name="placeholderText">
        <string>Password...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_ok">
       <property name="styleSheet">
        <string notr="true">QPushButton {
background-color: #404258;
color: #F5E8C7;
font-size: 18pt;
font-weight: 600;
margin: 10%;
padding: 15%;
border-radius: 28%;
}
QPushButton:hover {
background-color: #474E68;
}
QPushButton:pressed {
background-color: #50577A;
}</string>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
